falco/README.md

```diff
@@ -114,7 +114,7 @@ The following table lists the configurable parameters of the Falco chart and the
 | `falco.httpOutput.enabled`                      | Enable http output for security notifications                                                                      | `false`                                                                                                                                   |
 | `falco.httpOutput.url`                          | Url to notify using the http output when a notification arrives                                                    | `http://some.url`                                                                                                                         |
 | `falco.grpc.enabled`                            | Enable the Falco gRPC server                                                                                       | `false`                                                                                                                                   |
-| `falco.grpc.threadiness`                        | Number of threads (and context) the gRPC server will use                                                           | `8`                                                                                                                                       |
+| `falco.grpc.threadiness`                        | Number of threads (and context) the gRPC server will use, `0` by default, which means "auto"                                                           | `0`                                                                                                                                       |
```

Comment:
A leftover tab?

Reply:
Thanks for noticing this, I will fix it next time :)